helm/auto-deployment/configuration.md

```diff
@@ -0,0 +1,444 @@
+# Advanced Configuration
+
+## Client Libraries
+
+The following are the client libraries used in Bold BI by default.
+
+```console
+optionalLibs: 'mongodb,mysql,influxdb,snowflake,oracle,google,clickhouse'
+```
+
+Read the optional client library license agreement from the following link.
+
+[Consent to deploy client libraries](../docs/consent-to-deploy-client-libraries.md)
+
+If you wish to inlcude only specific client libraries note the optional client libraries from the above link as comma separated names in your values.yaml file.
+
+## Cluster Provider
+
+Bold BI supports Azure Kubernetes Service (AKS), Elastic Kubernetes Service (EKS) and Google Kubernetes Engine (GKE) you can enter the value based on your cluster provider.
+
+```console
+# your cluster provider name.The supported values are aks,gke and eks.
+clusterProvider: ''
+```
+
+
+## Persistent Volume
+
+### Name
+
+Persistent volumes were global resources. So if you already have Bold BI installed in your cluster, then the previous persistent volume name will conflict with current installation. Change this name to avoid conflicts with previous Bold BI persistent volumes.

```

Comment:
Persistent volumes were global resources. So if you have already installed Bold BI in your cluster, then the previous persistent volume name will conflict with the current installation. To avoid conflicts with previous Bold BI persistent volumes, change this name.

Reply:
concern addressed